client/web/src/enterprise/user/productSubscriptions/UserSubscriptionsProductSubscriptionsPage.tsx

```diff
@@ -88,8 +88,13 @@ export const UserSubscriptionsProductSubscriptionsPage: React.FunctionComponent<
                 path={[{ text: 'Subscriptions' }]}
                 description={
                     <>
-                        Contact us to purchase a subscription for a self-hosted Sourcegraph instance. See{' '}
-                        <Link to="https://about.sourcegraph.com/pricing">pricing</Link> for more information.
+                        <Text>
+                            Search your private code with{' '}
+                            <Link to="https://signup.sourcegraph.com/" onClick={() => eventLogger.log('ClickedOnCloudCTA')}>
+                                Sourcegraph Cloud
+                            </Link> or contact us to purchase a subscription for a self-hosted Sourcegraph instance. See{' '}
+                            <Link to="https://about.sourcegraph.com/pricing">pricing</Link> for more information.
+                        </Text>
```

Comment:
```suggestion
                        <>
                            Search your private code with{' '}
                            <Link to="https://signup.sourcegraph.com/" onClick={() => eventLogger.log('ClickedOnCloudCTA')}>
                                Sourcegraph Cloud
                            </Link> or contact us to purchase a subscription for a self-hosted Sourcegraph instance. See{' '}
                            <Link to="https://about.sourcegraph.com/pricing">pricing</Link> for more information.
                        </>
```
I think the `<>` fragment was used because of the console error below.  `<PageHeader/>` assigns `description` to a `Text` tag one level down
<img width="966" alt="image" src="https://user-images.githubusercontent.com/59381432/200682611-8f8da082-bbf0-420a-850d-8079784ac823.png">

Reply:
Ah, thanks Becca!